keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -0,0 +1,993 @@
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+the API proposed by this KEP aims to
+- Provide a reliable, consistent, and automated approach for any
+  multi-cluster application (framework, toolset) to discover available
+  clusters and take actions accordingly, in a way similar to service
+  discovery works in a microservice architecture. Through the inventory,
+  the application can query for a list of clusters to access, or watch
+  for an ever-flowing stream of cluster lifecycle events which the
+  application can act upon timely, such as auto-scaling, upgrades,
+  failures, and connectivity issues.
+- Provide a simple, clear interface for human operators to understand
+  clusters under management, and troubleshoot them in a streamlined
+  manner.
+
+### Terminology
+
+- **Cluster Manager**: An entity that creates the cluster inventory API 
+  resources per member cluster, and keeps their status up-to-date. Each
+  cluster manager SHOULD be identified with a unique name. Each cluster
+  inventory resource SHOULD be managed by only one cluster manager. Examples
+  of cluster manager are projects like OCM, Karmada, Clusternet or Azure
+  fleet manager.
+
+- **Cluster Inventory Consumer**: Users or third party tools that use
+  cluster inventory API for the purpose of workload distribution, operation
+  management etc. The consumer SHOULD not create this API resource.
+
+- **Member Cluster**: A kubernetes cluster that is managed by the cluster
+manager. A cluster manager SHOULD have sufficient permission to access
+the member cluster to fetch the information so it can update the status
+of the cluster inventory API resource.
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1: Multicluster Workload Distribution
+
+In this scenario, the unified API acts as a foundation for multicluster
+scheduling across various clusters. This means that the API will provide
+a standardized way to distribute workloads across multiple clusters. For
+instance, workload distribution tools like GitOps or
+[Work API](https://github.com/kubernetes-sigs/work-api) can leverage this
+API to make informed decisions about which cluster is best suited to handle
+a particular workload.
+
+This could be based on factors such as the current load on the cluster,
+its capacity, and its proximity to the data source or end-users. For
+example, if a cluster is already running at high capacity, the API could
+direct the workload to a less busy cluster. This ensures optimal
```

Comment:
As my comment above, this is not true. the API won't direct workloads. It provides information for Schedulers to make a decision. but the `ClusterInventory` CRD  is not a Scheduler API.  That would be MultiCluster-kueue or similar

Reply:
Thanks for pointing out. You're right. The API could provide useful information for the scheduler to make a precise decision, such as workload failover, re-balancing, etc.

Reply:
updated

Reply:
lgtm